core/src/main/scala/org/apache/spark/ui/WebUI.scala

```diff
@@ -193,3 +209,30 @@ private[spark] abstract class WebUIPage(var prefix: String) {
   def render(request: HttpServletRequest): Seq[Node]
   def renderJson(request: HttpServletRequest): JValue = JNothing
 }
+
+private[spark] class DelegatingServletContextHandler(servletContextHandler: ServletContextHandler) {
```

Comment:
`handler` is just as clear and much shorter as a name for the variable.

Reply:
i have renamed